README.md

```diff
@@ -23,8 +23,8 @@ Enables greater flexibility in creating journeys through the website
 | MONGODB_TOPICS_DATABASE      | topics                 | The MongoDB topics database
 | MONGODB_TOPICS_COLLECTION    | topics                 | MongoDB topics collection
 | MONGODB_CONTENT_COLLECTION   | content                | MongoDB content collection
-| MONGODB_USERNAME             | test                   | MongoDB Username
-| MONGODB_PASSWORD             | test                   | MongoDB Password
+| MONGODB_USERNAME             |                        | MongoDB Username
+| MONGODB_PASSWORD             |                        | MongoDB Password
```

Comment:
Should we provide defaults so people can run locally against a `mongodb` instance?

Reply:
Local instance of mongo db doesn't have the authorization by default.